src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+        icon: /images/activism-icon.svg
+tacoDiagramsInfo:
+  title: "Integrate a Privacy Layer with Nothing to Hide"
+  description: "TACo isn't just auditable, it's comprehensible, from high-level schematic to client code."
+  image: /images/taco-grid.svg
+  background: /images/glowing-background.png
+tacoDiagrams:
+  - title: "Generating a DKG Cohort"
+    preTitle: "learn"
+    description: "Independent nodes sampled from the Threshold network form a cohort – who collectively generate a DKG public key."
+    image: /images/first-taco-diagram.svg
+    buttons:
+      - label: "nucypher/ferveo"
+        url: https://github.com/nucypher/ferveo
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher-contracts"
+        url: https://github.com/nucypher/nucypher-contracts
+        variant: EXTERNAL_OUTLINE
+  - title: "Encrypting Private Data"
+    preTitle: "learn"
+    description: "Data owners specify conditions for decryption, embedded with private data and encrypted with the DKG public key."
+    image: "/images/second-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+  - title: "Accessing Decrypted Data"
+    preTitle: "learn"
+    description: "Requesters present state to the cohort, who verify conditions fulfillment before provisioning decryption material."
+    image: "/images/third-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+tacoExamplesInfo:
+  title: "Access conditions that fit your use case"
+  preTitle: "build"
+  description: "Whatever the business logic for data access, TACo nodes will enforce it. Specify and combine EVM, RPC and time-based conditions. Non-blockchain coming soon."
+  image: "/images/ecosystem-grid.svg"
+  rowReverse: false
+tacoExamples:
+  - title: "NFT-gating"
+    description: "Predicate decryption rights on the holding of special-purpose NFTs. Works for digital media, event streams and beyond."
+    image: "/images/nft-gating.png"
+  - title: "Secret recovery"
+    description: "Share your seed phrase or password  – to beneficiaries or your future self – without risking DOS by an intermediary."
+    image: "/images/secret-recovery.png"
+  - title: "Inference protection"
+    description: "Prevent third-parties from harvesting or blocking communication between end-users and generative models."
+    image: "/images/inference-protection.png"
+tacoTableInfo:
+  title: "The Zero Compromise Solution"
+  highlighted: ["Zero", "Compromise"]
+  description: "Choose a service that’s easy to integrate and that’s actually decentralized."
```

Comment:
```suggestion
  description: "Don’t choose between a service that’s easy to integrate and one that’s actually decentralized."
```

Reply:
I don't think I follow with the proposed suggestion. To me it would seem it's hinting that TACo is not easy to integrate.

Reply:
Mmm I thought this clearly implied that TACo doesn't require choosing between these two things, because you can have both at once.

Reply:
How about this? 

```suggestion
  description: "Adding extensions takes weeks. Building a battle-tested decentralized network takes years. Don't accept imitations."
```

Reply:
> Mmm I thought this clearly implied that TACo doesn't require choosing between these two things, because you can have both at once.

Perhaps the balance here with your original suggestion could be:
```suggestion
  description: "Use a service that offers both seamless integration and genuine decentralization, without sacrificing one for the other."
```

Reply:
I like this last suggestion but maybe we can go more succinct: 

```suggestion
  description: "Easy to integrate or genuinely decentralized. No longer a difficult choice."
```

Reply:
We could even make a 'trilemma' diagram here with 'Easy to Integrate', 'Programmable' and 'Decentralized'

Reply:
@evandrosaturnino Let's go with either: 
`Easy to integrate or genuinely decentralized. No longer a difficult choice.`
or 
`Adding extensions takes weeks. Building a battle-tested decentralized network takes years. Don't accept imitations.` 
@derekpierre do you have a preference?

Reply:
The question is that "Easy to integrate or genuinely decentralized. No longer a difficult choice." implies there is a choice to be made between both, but in fact the best way to put it that TACo is both easy to integrate and decentralized

Reply:
@arjunhassard Admittedly, I have a preference for an explicit statement instead of an implicit statement, leaving the user to interpret it, i.e. something closer to what @evandrosaturnino stated, so something like the above or to blend both:
`TACo combines ease of integration and genuine decentralization. No need to choose.` / 
`TACo blends ease of integration with genuine decentralization. No concession required.`

But if you feel strongly about it, I can be on board with the following:
`Easy to integrate or genuinely decentralized. No longer a choice.`

Reply:
Thanks, I agree with both of you. It's not perfect, but let's go with: 

```suggestion
  description: "TACo blends ease of integration with genuine decentralization. No need to choose between the two."
```